Testnet/NonFungibleToken/NonFungibleToken/INonFungibleTokenMetadata.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace NonFungibleTokenContract
+{
+    public interface INonFungibleTokenMetadata
+    {
+        string Name { get; }
+        string Symbol { get; }
+
+        string TokenURI(ulong tokenId);
+    }
+}
```

Comment:
Is this really even used anywhere? 

Should the namespace be included? Should the interface be inline with the rest of the compiled contract code?

Reply:
These interfaces are a kind of documentation or standardization of current NFT contract. And it is not harmful to keep it in repository. Actually these interfaces are not included compiled bytecode. 

We should to move these interfaces into Stratis.SmartContracts.Standards library in future.